libtransact/Cargo.toml

```diff
@@ -66,5 +63,17 @@ contract-address-triple-key-hash = ["contract-address"]
 contract-context = ["contract", "contract-address"]
 contract-context-key-value = ["contract-context", "key-value-state"]
 key-value-state = []
+redis-db = ["redis"]
+sawtooth-compat = ["sawtooth-sdk"]
 sqlite-db = ["rusqlite", "r2d2", "r2d2_sqlite"]
 state-merkle = ["cbor-codec"]
+ursa-compat = ["ursa"]
+
+[package.metadata.docs.rs]
+features = [
+  "stable",
+  "experimental",
+  "sawtooth-compat",
+  "state-merkle",
+  "ursa-compat",
```

Comment:
I think this is going to break the docs on publish.

Reply:
Removed ursa from the list